packages/core/stories/patterns/app-header/app-header.stories.tsx

```diff
@@ -0,0 +1,227 @@
+            <StackLayout
+              direction="row"
+              style={{
+                width: "100%",
+                backgroundColor: "var(--salt-container-primary-background)",
+                zIndex: "calc(var(--salt-zIndex-drawer) + 1)",
+                position: "fixed",
+                inset: "0 0 auto 0",
+                paddingRight: "1em",
+                borderBottom:
+                  "var(--salt-size-border) var(--salt-container-borderStyle) var(--salt-container-secondary-borderColor)",
+              }}
+            >
+              <FlexItem
+                style={{
+                  justifyContent: "center",
+                  display: "flex",
+                  height:
+                    "calc(var(--salt-size-base) + var(--salt-spacing-200))",
+                  width:
+                    "calc(var(--salt-size-base) + var(--salt-spacing-200))",
+                  borderRight:
+                    "var(--salt-size-border) var(--salt-container-borderStyle) var(--salt-container-secondary-borderColor)",
+                }}
+              >
+                <Button
+                  onClick={handleDrawerToggle}
+                  variant={drawerOpen ? "primary" : "secondary"}
+                  style={{ alignSelf: "center" }}
+                >
+                  {drawerOpen ? <CloseIcon /> : <MenuIcon />}
+                </Button>
+              </FlexItem>
+              <FlexItem align="center">App logo</FlexItem>
+            </StackLayout>
+            <Drawer
+              style={{
+                paddingTop:
+                  "calc(var(--salt-size-base) + var(--salt-spacing-200))",
+                paddingLeft: "0",
+              }}
+              open={drawerOpen}
+            >
+              <nav>
+                <ul style={{ listStyle: "none", padding: "0" }}>
+                  {items?.map((item) => (
+                    <li key={item}>
+                      <NavigationItem
+                        orientation="vertical"
+                        active={active === item}
+                        href="#"
+                        onClick={() => setActive(item)}
+                      >
+                        {item}
+                      </NavigationItem>
+                    </li>
+                  ))}
+                  {utilities?.map((utility) => (
+                    <li key={utility.key}>
+                      <NavigationItem orientation="vertical" href="#">
+                        {utility.icon}
+                        {utility.label}
+                      </NavigationItem>
+                    </li>
+                  ))}
+                </ul>
+              </nav>
+              <StackLayout
+                align="start"
+                direction="column"
+                gap={1}
+                style={{
+                  paddingLeft:
+                    "calc(var(--salt-spacing-300) + var(--salt-spacing-25))",
+                }}
+              ></StackLayout>
```

Comment:
is this meant to wrap somethig?

Reply:
oops it was but I moved something around